certified-connectors/Iconfirm/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4257 @@
+          "format": "int32",
+          "description": "The interval amount of months for when revisions should be done on the system/recipient.",
+          "type": "integer"
+        },
+        "revisionIntervallSubprocessor": {
+          "format": "int32",
+          "description": "The interval amount of months for when revisions should be done on the system/recipient subprocessor(s).",
+          "type": "integer"
+        },
+        "dataBreachNotificationTime": {
+          "format": "int32",
+          "description": "In case of data breach, deadline for notification to the controller (hours).",
+          "type": "integer"
+        },
+        "subprocessors": {
+          "description": "Subprocessors associated with the system/recipient.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FullSubprocessorObject"
+          }
+        },
+        "organizationalLink": {
+          "description": "Which part of the organisation uses the solution.",
+          "type": "string"
+        },
+        "allowRecipientToUseOwnOrganizationSubprocessors": {
+          "description": "If the use of sub-processors within Processors own Group of companies should be allowed.",
+          "type": "boolean"
+        },
+        "mainContract": {
+          "description": "The name of the main contract(s) the system/recipient.",
+          "type": "string"
+        },
+        "mainContractSignedAt": {
+          "format": "date-time",
+          "description": "The date of then the main contract for the system/recipient was signed.",
+          "type": "string"
+        },
+        "controllerSignee": {
+          "$ref": "#/definitions/SigneeContact"
+        },
+        "recipientSignee": {
+          "$ref": "#/definitions/SigneeContact"
+        },
+        "dataBreachContact": {
+          "$ref": "#/definitions/Contact"
+        },
+        "revisionsOfRecipientNotApplicateble": {
+          "description": "If revisions of the recipient isn't applicateble.",
+          "type": "boolean"
+        },
+        "revisionsOfSubprocessorsNotApplicable": {
+          "description": "If revisions of the recipients subprocessor(s) isn't applicateble.",
+          "type": "boolean"
+        },
+        "purposes": {
+          "description": "The purposes registered on the system/recipient for its nature of processing.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PurposeObject"
+          }
+        }
+      }
+    },
+    "TransportType": {
+      "description": "The different communication channels",
+      "enum": [
+        "Email",
+        "SMS",
+        "Annotation"
+      ],
+      "type": "string"
+    },
+    "Treatment": {
+      "enum": [
+        "Avoid",
+        "Transfer",
+        "RC",
+        "Accept"
+      ],
+      "type": "string"
+    }
+  },
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "API-n\u00f8kkel": {
```

Comment:
@iconfirm-development Unicodes - The connector's swagger still have unicodes which may not be supported in all the Power Automate page, Can you please replace all the unicodes with respective characters in the English language.

Reply:
swagger import miss, updated from norwegian 'ø' to API-Key now

Reply:
@iconfirm-development And non-English characters are not allowed in any connector artifacts.